cloudbuild.yaml

```diff
@@ -28,7 +28,7 @@ steps:
   # The following step starts the build process using Gradle official image, performs the build
   # and runs `jib` for the specified `gcpProject` in order to build and deploy the container
   # image to the Google Container Registry.
-  - name: 'gradle:6.8-jdk11'
+  - name: 'gradle:6.9-jdk11'
```

Comment:
Did you try 7.2?

Reply:
Yep. Spine 1.x (bootstrap plugin) is not compatible with Gradle 7.x. I plan to migrate the app to Spine 2.x and do the migration to Gradle 7.x there.